configuration/src/main/java/pl/edu/uj/kimage/configuration/SettingName.java

```diff
@@ -0,0 +1,5 @@
+package pl.edu.uj.kimage.configuration;
+
+public enum SettingName {
+    NUMBER_OF_THREADS
```

Comment:
It's better than keep everywhere uncontrolled Strings or create a class per setting (class explosion)